commit 3c7cfd074d3783bc37b00ecc451fbf749956bc17 (refs/changes/48/4448/1)
Author: Tim Foster <tim.foster@joyent.com>
Date:   2018-06-27T16:04:56+01:00 (1 year, 3 months ago)
    
    TRITON-552 components using cp -P during their build fail with /usr/bin/cp

diff --git a/Makefile b/Makefile
index 8aaad6d..e243f30 100644
--- a/Makefile
+++ b/Makefile
@@ -97,7 +97,7 @@ service: all $(SMF_MANIFESTS)
 	    $(TOP)/build/service/root/opt/smartdc/boot/
 	cp -R $(TOP)/boot/* \
 	    $(TOP)/build/service/root/opt/smartdc/boot/
-	cp -P smf/manifests/sapi.xml $(SVC_INSTDIR)/smf/manifests
+	cp smf/manifests/sapi.xml $(SVC_INSTDIR)/smf/manifests
 	cp -r $(TOP)/sapi_manifests $(SVC_INSTDIR)/
 	cp -r $(TOP)/test $(SVC_INSTDIR)/
 	cp -PR $(NODE_INSTALL) $(SVC_INSTDIR)/build/node
